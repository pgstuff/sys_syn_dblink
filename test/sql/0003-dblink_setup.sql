BEGIN;

CREATE EXTENSION sys_syn;

CREATE EXTENSION dblink;

CREATE EXTENSION sys_syn_dblink;

SELECT  dblink_connect('sys_syn_dblink_test', 'dbname=contrib_regression host=' ||
        quote_literal(split_part((SELECT pg_settings.setting FROM pg_settings WHERE pg_settings.name = 'unix_socket_directories'), ', ', 1)));

SELECT dblink_disconnect('sys_syn_dblink_test');

ROLLBACK;
