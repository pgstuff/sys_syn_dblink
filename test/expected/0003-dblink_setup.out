BEGIN;
CREATE EXTENSION pgcrypto;
CREATE EXTENSION sys_syn;
CREATE EXTENSION hstore;
CREATE EXTENSION dblink;
CREATE EXTENSION sys_syn_dblink;
SELECT  dblink_connect('sys_syn_dblink_test', 'dbname=contrib_regression host=' ||
        quote_literal(split_part((SELECT pg_settings.setting FROM pg_settings WHERE pg_settings.name = 'unix_socket_directories'), ', ', 1)));
 dblink_connect 
----------------
 OK
(1 row)

SELECT dblink_disconnect('sys_syn_dblink_test');
 dblink_disconnect 
-------------------
 OK
(1 row)

ROLLBACK;
